src/components/ScholarshipDetailCard.jsx

```diff
@@ -118,52 +118,63 @@ export default function ScholarshipDetailCard({ scholarship }) {
     text: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
     bottom: PropTypes.bool,
     top: PropTypes.bool,
+    link: PropTypes.bool,
   };
   DetailCardCell.defaultProps = {
     bottom: false,
     top: false,
+    link: false,
   };
 
   return (
     <Card className={classes.detailSection}>
       <Typography variant="h4">{name}</Typography>
-      <Typography variant="h5">{organization}</Typography>
+      <Typography variant="h5" gutterBottom>
+        {organization}
+      </Typography>
+
       <Box className={classes.actionSection}>
         <Button
           component={Link}
           href={website}
           variant="contained"
           color="primary"
           className={classes.applyBtn}
-          startIcon={<Send />}>
+          startIcon={<Send />}
+          disabled={scholarship.id === undefined}>
```

Comment:
This button isn't dependent on the ID, it uses the scholarship website URL. You could set [`target`](https://www.w3schools.com/tags/att_a_target.asp) to `_blank` though so that clicking on it opens a new tab instead of opening it in the current tab (this is probably a fine default behavior to have for the ViewScholarship case too).

Reply:
nice, made the changes. Now when clicked, it opens on another tab.